wordpress-mobile/WordPress-iOS
# [Stats] Update ghost cell colors for Dark Mode

Ref #12405 

This updates the colors used in the Stats ghost cells.

To test:
- On a slow network, go to Stats > Insights.
- Verify the ghost colors are correct in dark mode.

![ghost_colors](https://user-images.githubusercontent.com/1816888/66523878-2f487e80-eaae-11e9-94e9-2228dd1fbc14.gif)

Update release notes:
